src/utils/url.ts

```diff
@@ -0,0 +1,24 @@
+// util function that generates a url string from a base url and a query object
```

Comment:
So, this will generate the url that targets the object's resource on the backend just from getting the object?

Reply:
Exactly, there's a general endpoint for fetching lists or POSTing a new one, but for editing or deleting a specific object we need the specific one

Reply:
Wow great idea on that.